### PR TITLE
Tiny Fixes

### PR DESCRIPTION
**Pull request by ghost**
Not really much but honest work type of edits. (And a free typo check)

### CODE REVIEW

**Comment by AJMC-DEV** [comment]
Lmao thanks forgi

**Review by AJMC-DEV — APPROVED** [review]

File: events/message.js [hunk 1]
```diff
@@ -9,25 +9,23 @@ module.exports = async (boat, message) => {
   if (message.channel.type !== 'text') return;
 
   if (!message.content.startsWith(boat.prefix)) {
-    handleRaft(boat.rafts, message);
-    return;
+    return handleRaft(boat.rafts, message);
```
Comment (ckohen) [inline events/message.js]: This fails eslint (will add github action soon:tm:)

File: events/message.js [hunk 2]
```diff
@@ -9,25 +9,23 @@ module.exports = async (boat, message) => {
...
   }
 
   const args = message.content.slice(boat.prefix.length).trim().split(/\s+/g);
   const command = args.shift().toLowerCase();
 
   const handler = boat.commands.get(command);
   if (!handler) {
-    handleRaft(boat.rafts, message);
-    return;
+    return handleRaft(boat.rafts, message);
```
Comment (ckohen) [inline events/message.js]: Same

**Review by ckohen — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Re: One-liner failing ESLint 
I reverted the code back to the normal and non-ESLint-failing state

**Review by ckohen — APPROVED** [review]

**Pull request merged**